tests/decode/test2_suit/CMakeLists.txt

```diff
@@ -31,3 +31,7 @@ include(${PROJECT_BINARY_DIR}/manifest2.cmake)
 
 target_link_libraries(manifest2 PRIVATE zephyr_interface)
 target_link_libraries(app PRIVATE manifest2)
+
+if (NOT VERBOSE)
+  target_compile_options(manifest2 PRIVATE -Wpedantic -Wconversion)
+endif()
```

Comment:
Imo all of these test cmake files should `include` a common file which did all the common configurations like this.

Reply:
It's not in all tests, and it requires the name of the library, so I'd rather keep it this way.